services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HaittaFactory.kt

```diff
@@ -0,0 +1,86 @@
+package fi.hel.haitaton.hanke.factory
+
+import fi.hel.haitaton.hanke.domain.Haittojenhallintasuunnitelma
+import fi.hel.haitaton.hanke.domain.Haittojenhallintatyyppi
+import fi.hel.haitaton.hanke.tormaystarkastelu.Autoliikenneluokittelu
+import fi.hel.haitaton.hanke.tormaystarkastelu.AutoliikenteenKaistavaikutustenPituus
+import fi.hel.haitaton.hanke.tormaystarkastelu.HaittaAjanKestoLuokittelu
+import fi.hel.haitaton.hanke.tormaystarkastelu.Liikennemaaraluokittelu
+import fi.hel.haitaton.hanke.tormaystarkastelu.TormaystarkasteluKatuluokka
+import fi.hel.haitaton.hanke.tormaystarkastelu.TormaystarkasteluTulos
+import fi.hel.haitaton.hanke.tormaystarkastelu.VaikutusAutoliikenteenKaistamaariin
+
+object HaittaFactory {
+
+    val TORMAYSTARKASTELU_DEFAULT_AUTOLIIKENNELUOKITTELU =
+        Autoliikenneluokittelu(
+            HaittaAjanKestoLuokittelu.YLI_KOLME_KUUKAUTTA.value,
+            TormaystarkasteluKatuluokka.TONTTIKATU_TAI_AJOYHTEYS.value,
+            Liikennemaaraluokittelu.LIIKENNEMAARA_ALLE_500.value,
+            VaikutusAutoliikenteenKaistamaariin.YKSI_KAISTA_VAHENEE.value,
+            AutoliikenteenKaistavaikutustenPituus.PITUUS_ALLE_10_METRIA.value,
+        )
+
+    val TORMAYSTARKASTELU_ZERO_AUTOLIIKENNELUOKITTELU =
+        Autoliikenneluokittelu(
+            1,
+            0, // if both 'katuluokka' and 'liikennemaara' is 0, the index is 0.0
+            0, // if both 'katuluokka' and 'liikennemaara' is 0, the index is 0.0
+            1,
+            1,
+        )
+
+    val DEFAULT_HHS_YLEINEN = "Yleisten haittojen hallintasuunnitelma"
```

Comment:
These Strings could be consts.

Reply:
Yes, thank you. Changed these to constants.